### ISSUE TITLE
"uno mil" -> "un mil"

### ISSUE THREAD

**Issue by zarza** [yamadapc/js-written-number#74]
Hello!

There is a small issue in spanish because it says "uno mil" instead of "un mil".

For example:
https://jsfiddle.net/zarza/un4swL1h/3/

Result:
catorce millones novecientos ochenta y **uno mil** cuatrocientos cincuenta y seis

And it should be just like NumberFormat in PHP:
catorce millones nove­cientos ochenta y **un mil** cuatrocientos cincuenta y seis

PHP code:
```
$formatter = new \NumberFormatter('es', \NumberFormatter::SPELLOUT);
echo $formatter->format(14981456) . "\n";

```
Any idea of how can we improve this? Currently I'm doing a simple replace from "uno mil" to "un mil".

**Comment by pexposito** [comment]
Hi @zarza, I allready created the same issue, #72 , and made a pull request to solve this. Meanwhile, you can override de spanish configuration to add "useBaseExceptionWhenNoTrailingNumbers": true at units configuration, like this:

`
"units" : [
    {
      "singular": "ciento",
      "useBaseInstead": true,
      "useBaseException": [1]
      "useBaseException": [1],
      "useBaseExceptionWhenNoTrailingNumbers": true
    },
`

**Comment by forzagreen** [comment]
Hi @zarza & @pexposito , we are working on a new project https://github.com/forzagreen/n2words . It's already supporting 20 languages and still growing.

@zarza do you know where I can find the source code of the PHP NumberFormat ?

**Comment by yamadapc** [comment]
Fixed in v0.11.1
